### ISSUE TITLE
Unable to Query Parameters containing '#' in XrmQuery

### ISSUE THREAD

**Issue by krihvi** [delegateas/XrmDefinitelyTyped#78]
Using the following code:
![image](https://user-images.githubusercontent.com/18004958/46731579-5b5ad500-cc8b-11e8-97a6-e0f392f0ca31.png)

with the following user:
`Test 04 #`

Result in:
```
dg.xrmquery.web.promise.min.js:1 
GET https://crmurl/api/data/v8.0/systemusers
$select=domainname&$filter=fullname%20eq%20%27Test%2004%20 
400 (Bad Request)
```

It appears that the query cuts off when it comes to the hashtag.
If %23%27 is added to the request will the request run as intended.

**Comment by krihvi** [comment]
I discovered that we are using version 2.5.1. I will try to update to 4.0.3 and will try to recreate the issue.
Will update on the findings

**Comment by majakubowski** [comment]
This bug was reported in issue #59 and fixed in pull request #60 .. works fine from version 2.6.2 :)

**Comment by magesoe** [comment]
Closing this issue based upon the answer from @majakubowski, let me know if it still persists.
